practice-app/api/views.py

```diff
@@ -1,3 +1,69 @@
+from django.http.response import HttpResponse
 from django.shortcuts import render
+import requests
+from django.views.decorators.csrf import csrf_exempt
+from .models import Story
+from django.http import JsonResponse
+import json
+import environ
 
 # Create your views here.
+env = environ.Env(DEBUG=(bool, False))
+
+WEATHER_API_KEY=env('WEATHER_API_KEY')
+
+@csrf_exempt
+def weather(request,story_id):
+
+    if(request.method!='GET'):
+        httpresponse=HttpResponse('Only GET method is available for this API')
+        httpresponse.status_code=400
```

Comment:
I think status code 405 could be more appropriate here.

Reply:
You are right. I edited that part. Thank you!